lms/static/js/spec/learner_dashboard/collection_list_view_spec.js

```diff
@@ -29,7 +29,14 @@ define([
                             modified: '2016-03-25T13:45:21.220732Z',
                             marketing_slug: 'p_2?param=haha&test=b', 
                             id: 146,
-                            marketing_url: 'http://www.edx.org/xseries/p_2?param=haha&test=b'
+                            marketing_url: 'http://www.edx.org/xseries/p_2?param=haha&test=b',
+                            /* jshint ignore:start */
+                            banner_image_urls: [
+                                '348x116': 'http://www.edx.org/images/org1/test1',
+                                '435x145': 'http://www.edx.org/images/org1/test2',
+                                '726x242': 'http://www.edx.org/images/org1/test3'
+                            ]
+                            /* jshint ignore:end */
```

Comment:
@schenedx I don't think these need to be numbers, and if jshint is throwing an error it probably makes sense to update the code to used names instead of number values. Why not just use ```small, medium, large``` like used in the rest of the code?